src/test/shared/sam/debugger/awsSamDebugConfigurationProvider.test.ts

```diff
@@ -142,20 +142,6 @@ describe('AwsSamDebugConfigurationProvider', async () => {
             assert.strictEqual(resolved, undefined)
         })
 
-        it('returns undefined when resolving debug configurations with an invalid target type', async () => {
```

Comment:
The type system won't allow this test to compile.

Reply:
Here's a gross way to sidestep the issue, change line 151 to:
                    target: 'not-code' as 'code',

Reply:
@awschristou see https://github.com/aws/aws-toolkit-vscode/pull/967#discussion_r394650095 for my thoughts on this

Reply:
Let me see how the test behaves if I do this.

Reply:
TECHNICALLY that won't fit the spirit of this test, this is specifically testing an invalid target, not a mismatched target.

Reply:
so ... we're fine deleting it? and relying on the type-system?

Reply:
I'd like confirmation that VS Code blocks all non-'code' or 'template'  targets from making it through to this function, otherwise the only safe way is to have target be typed as a string (and let the code handle it)